geocropper/utils.py

```diff
@@ -1837,3 +1837,117 @@ def move_crops_containing_locations(csv_path, source_dir, target_dir):
                             target_dir.mkdir(parents=True, exist_ok=True)
                             shutil.move(str(crop.absolute()), str(target_dir.absolute()))
                             break                 
+
+
+def get_unique_lat_lon(root_dir, speckle):
+    """Loops through every folder of the specified speckle variant in rootdir and returns 
+    a set with unique latitude and longitude positions.
+
+    Parameters
+    ----------
+    root_dir: Path
+        Path of trimmed crops with various speckle variants and recording times.
+    speckle: String
+        Set desired speckle variant (options available in geocropper.stack_trimmed_images) 
+    """
+    
+    # Set is an unordered list which allows no duplicated entries
+    lat_lon_set = set()
+
+    print(f"Reading unique positions for {speckle}...")
+
+    for folder in root_dir.glob(f"*{speckle}*"):

```

Comment:
For postfix it would be this then (without second asterix):
`for folder in source_dir.glob(f"*{postfix}"):`